src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -268,7 +266,7 @@ void StartBulkFileOperationNotification()
                     // so guarding us from them
                     if (localRegistration != null)
                     {
-                        WatsonReporter.Report(new Exception("BulkFileOperation already exist"));
+                        FatalError.ReportWithoutCrash(new InvalidOperationException("BulkFileOperation already exist"));
```

Comment:
was this necessary to change the exception type?  (it's not a problem, just curious).

Reply:
It wasn't necessary. It's generally good practice to throw more specific exception types.